common/components/common/event_projects/JoinConferenceButton.jsx

```diff
@@ -32,10 +32,6 @@ class JoinConferenceButton extends React.PureComponent<Props> {
         >
           <i className={Glyph(GlyphStyles.Video, GlyphSizes.LG) + " mr-3"} />
           {this.props.children}
-          <i className={Glyph(GlyphStyles.Users, GlyphSizes.LG) + " ml-3"} />
-          <span className="JoinConference-usercount overline ml-1">
-            {this.props.participant_count || "0"}
-          </span>
```

Comment:
This looks like the only applicable change related to this feature.  Is there something I'm missing with the other changes?